### DISCUSSION TITLE
links

### DISCUSSION THREAD

**Discussion by nihat961** [community/community#137131] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Congratulations on building a great browser. When I click on any link, it does not open on the side. The link I clicked opens in the same place. What do I need to do to adjust this? Is there a plugin or something?

**Comment by Yackens** [comment]
It depends on the browser you are using.

**Chrome:**
**Middle Click or Ctrl + Click:** You can middle-click or hold Ctrl while clicking on a link to open it in a new tab.
**Settings**: Go to Settings > On startup, and you can choose to continue where you left off to keep tabs open after a session.

**Firefox**:
**Middle Click or Ctrl + Click**: Same as in Chrome.
**Preferences**: In the settings, go to Preferences > Tabs and ensure that "Open links in tabs instead of new windows" is checked.

**Microsoft Edge**:
**Ctrl + Click**: Hold down the Ctrl key while clicking a link to open it in a new tab.
**Shift + Click**: Hold down the Shift key while clicking a link to open it in a new window.
**Settings**: In Settings, go to System and Performance. Enable "Continue running background apps when Microsoft Edge is closed" and "Continue where you left off" under the On startup section.

If you found this answer useful, don't forget to mark it as the accepted answer so more people can see it.

**Comment by ebndev** [comment]
Hi @nihat961, thanks for being a part of the GitHub Community!

Can you share a few more details about what link or page you're talking about? That might help someone give you a nudge in the right direction. 😄
